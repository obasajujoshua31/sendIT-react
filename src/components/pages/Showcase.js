import React from "react";
import PageFooter from '../shared/PageFooter';
import AccountContent from "../shared/AccountContent";
const Showcase = () => {
  return (
    <div>
      <div className="showcase">
        <div className="container user-information">
          <AccountContent />
          {/* <PageFooter/> */}
          
        </div>
      </div>
    </div>
  );
};

export default Showcase;
