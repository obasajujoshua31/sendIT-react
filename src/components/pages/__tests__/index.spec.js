import React from "react";
import { shallow } from "enzyme";
import HomePage from "../Homepage";
import Showcase from "../Showcase";

describe("<HOmePage>", () => {
  const wrapper = shallow(<HomePage />);
  it("should render Home page", () => {
    expect(wrapper).toMatchSnapshot();
  });
  it("should contain 1 <div>", () => {
    expect(wrapper.find("div").length).toEqual(1);
  });
});

describe("<Show Case>", () => {
  const wrapper = shallow(<Showcase />);
  it("should render show case component", () => {
    expect(wrapper).toMatchSnapshot();
  });
  it("should contain 3 divs", () => {
    expect(wrapper.find("div").length).toEqual(3);
  });
});
