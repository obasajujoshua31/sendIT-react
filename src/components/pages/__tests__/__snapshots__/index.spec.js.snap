// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HOmePage> should render Home page 1`] = `
<div>
  <Showcase />
</div>
`;

exports[`<Show Case> should render show case component 1`] = `
<div>
  <div
    className="showcase"
  >
    <div
      className="container user-information"
    >
      <withRouter(AccountContent) />
    </div>
  </div>
</div>
`;
